docker/pypi/reqmgr2ms-unmerged/run.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+# script to start MSUnmerged
+
+
+### Usage: Usage: run.sh [-e 'RSE Expression']
+### Usage:
+### Usage:   -e  <RSE Expression>   A Rucio RSE Expression to be passed to the service configuration
+### Usage:                          before service start
+### Usage:   -h <help>              Provides help to the current script
+### Usage:
+### Usage: Example: ./run.sh -e 'rse_type=DISK&country=US&tier=3&cms_type=real'
+### Usage: Example: ./run.sh
+### Usage:
+
+FULL_SCRIPT_PATH="$(realpath "${0}")"
+
+usage()
+{
+    echo -e $*
+    grep '^### Usage:' < $FULL_SCRIPT_PATH
+    exit 1
+}
+
+help()
+{
+    echo -e $*
+    grep '^### Usage:' < $FULL_SCRIPT_PATH
+    exit 0
+}
+
+### Searching for the mandatory and optional arguments:
+# export OPTIND=1
+while getopts ":e:h" opt; do
+    case ${opt} in
+        e)
+            rseExpr=${OPTARG}
+            # escaping all possible special characters in the RSE expression:
+            # NOTE: One should always start escaping the escaping chars first
+            rseExpr=${rseExpr//\\/\\\\}
+            rseExpr=${rseExpr//&/\\&}
+            rseExpr=${rseExpr//|/\\|}
+            ;;
+        h)
+            help
+            ;;
+        \? )
+            msg="Invalid Option: -$OPTARG"
+            usage "$msg"
+            ;;
+        : )
+            msg="Invalid Option: -$OPTARG requires an argument"
+            usage "$msg"
+            ;;
+    esac
+done
+
+
+srv=`echo $USER | sed -e "s,_,,g"`
+STATEDIR=/data/srv/state/$srv
+LOGDIR=/data/srv/logs/$srv
+AUTHDIR=/data/srv/current/auth/$srv
+CONFIGDIR=/data/srv/current/config/$srv
+CONFIGFILE=${CONFIGFILE:-config.py}
+CFGFILE=/etc/secrets/$CONFIGFILE
+
+### permission update to workaround issues with mounting logs volume
+sudo chown -R $USER.$USER /data
```

Comment:
Just an observation for the future, this command is already executed in the Dockerfile and it takes a few minutes to complete. In the future, we need to have a solution that will not delay the startup of the service.